documentation/book/assembly-custom-resources.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// assembly-deployment-configuration.adoc
+
+[id='assembly-custom-resources-{context}']
+= Custom resources
+Custom Resource Definitions (CRDs) extend the Kubernetes API, providing definitions to create and add custom resources to an {ProductPlatformName} cluster. Custom resources are created as instances of CRDs.
+
+For {Productname}, CRDs introduce custom resources specific to {Productname} to an {ProductPlatformName} cluster, such as Kafka, Kafka Connect, Kafka Mirror Maker, and users and topics.  CRDs provide configuration instructions, defining the schemas used to instantiate and manage the resources specific to {Productname}. CRDs also allow {Productname} resources to benefit from native {ProductPlatformName} features like CLI accessibility and configuration validation.
```

Comment:
* It's `{ProductName}` (capital N) I think.
* "For {Productname}, CRDs introduce custom resources specific to {Productname}" – too many {Productname} in that perhaps?
* "instantiate and manage the resources" → "instantiate and manage the custom resources" ?

Note: sentence-per-line would make it much easier to use suggested edits for changes like this. One long line makes it more likely a reviewer will raise multiple unrelated points and less likely they can just accept all the suggested edits. The choice is up to you, but it's you who stands to benefit from reviewers using the suggested edit feature.

Reply:
Thanks Tom. Built locally with the attribute, but it's as you say. Updated. 

I'll bear in mind what you say about single sentence lines.